src/main/java/com/internet/shop/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.dao.ShoppingCartDao;
+import com.internet.shop.library.Inject;
+import com.internet.shop.library.Service;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Inject
+    private ProductDao productDao;
```

Comment:
why you inject productDao here if you don't you use it

Reply:
redundant import, thanks